docs/proposals/docker-registry.md

```diff
@@ -0,0 +1,33 @@
+## Improve docker-registry
+
+We need to stop using `linkedBlobStore` from `docker/distribution`. The main purpose for this module is to store information
```

Comment:
I'm not sure this problem statement has enough detail for someone not familiar with the registry.  I would include info on why we even have a linked blob store and the functionality it provides.  Also, please include information about why you're proposing a refactor.  What benefit does it give us if the linkedBlobStore is already providing the functionality.  (it is sort of mentioned in the second paragraph here but I think calling it out but there is no justification on why it helps us)

Take a look at some of the existing proposals in https://github.com/openshift/origin/tree/master/docs/proposals.